cmd/zfs/zfs_main.c

```diff
@@ -8114,6 +8132,12 @@ main(int argc, char **argv)
 	    (strcmp(cmdname, "--help") == 0))
 		usage(B_TRUE);
 
+	/*
+	 * Special case '-V|--version'
+	 */
+	if ((strcmp(cmdname, "-V") == 0) || (strcmp(cmdname, "--version") == 0))
+		(void) zfs_do_version(argc, argv);
```

Comment:
`return (zfs_do_version(argc, argv));` with the changes above.

Reply:
done